src/main/scala/com/touk/hiring_task/multiplex/model/Cinema.scala

```diff
@@ -0,0 +1,49 @@
+package com.touk.hiring_task.multiplex.model
+
+import com.touk.hiring_task.multiplex.model.BookingStatus.BookingStatus
+import com.touk.hiring_task.multiplex.model.RoomName.RoomName
+import com.touk.hiring_task.multiplex.model.TicketType.TicketType
+
+import scala.language.implicitConversions
+import scala.util.Try
+
+case class User(name: String)
+
+case class Place(row: Int, seat: Int)
+
+case class Room(name: RoomName, row: Int = 5, placeInRow: Int = 5)
+
+case class Projection(id: String, room: Room, startAt: Long, title: String)
+
+case class Reservation(user: User, ticket: TicketType, status: BookingStatus, expireAt: Long)
+
+case class Key(projectionId: String, roomName: String, place: Place)
+
+object RoomName extends Enumeration {
```

Comment:
Do you know why Enumeration is not the best choice in Scala? Do you know some other alternatives?